src/config.json

```diff
@@ -42,6 +42,13 @@
         "markdown": true
       }
     },
-    "faviconPath": "assets/img/gnosis_logo_icon.svg"
+    "faviconPath": "assets/img/gnosis_logo_icon.svg",
+    "provider": "UPORT",
```

Comment:
Is this a `defaultProvider` maybe?

Reply:
@andre-meyer For me "defaultProvider" implies that there will be more providers than just one. Here we'll use the only provider specified. hope you get it